documentation/source/simulator_support.rst

```diff
@@ -7,6 +7,8 @@ This page documents any known quirks and gotchas in the various simulators.
 Icarus
 ------
 
+vector bug
```

Comment:
vector bug -> Accessing bits in a vector

(Avoid calling it a bug, it's possibly just a missing feature.)